website/docs/reference/dbt-jinja-functions.md

```diff
@@ -26,6 +26,7 @@ In addition to the standard Jinja library ([docs](https://jinja.palletsprojects.
 * [model](model)
 * [modules](modules)
 * [on-run-end-context](on-run-end-context)
+/<VersionBlock firstVersion="1.1">* [print](print)</VersionBlock>
```

Comment:
@JKarlavige Is there a way to make this work for versioning one line? When I use the front slash, the link works but it's not bulleted.

Reply:
This is a good question! It looks like this will require a small adjustment to the VersionBlock component.

I'm creating an Asana task and will open a separate PR to get that fix in. In the meantime, adjust this line to read as follows:

`* &nbsp;<VersionBlock firstVersion="1.1">[print](print)</VersionBlock>`

The `&nbsp;` is required for using the VersionBlock component at the start of a new line. And the adjustment I will push through will be changing the VersionBlock from wrapping the content within it from a div, to a react fragment. The div is breaking it in this case.